CHANGELOG.md

```diff
@@ -4,6 +4,168 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [4.1.1] - TBD
+
+See the [release announcement](https://godotengine.org/article/maintenance-release-godot-4-1-1) for details.
+
+### Added
+
+#### Animation
+
+- Add 3.x compatibility for animation loop mode ([GH-79155](https://github.com/godotengine/godot/pull/79155)).
+
+#### C#/.NET
+
+- Automatically generate version defines ([GH-78722](https://github.com/godotengine/godot/pull/78722)).
```

Comment:
This is just a buildsystem internal change that has no effect on users compared to a PR manually updating the version, so I wouldn't feature this so prominently, if at all.
Ie for users this is basically on the same level as increasing a version number.